pnpxai/explainers/lear.py

```diff
@@ -0,0 +1,43 @@
+import torch
+from transformers import AutoModel
+from pnpxai.explainers.base import Explainer
+
+
+class LEAR(Explainer):
+    """
+    LEAR Explainer class for generating counterfactual explanations.
+
+    Args:
+        model_name (str): The name of the pre-trained model to load.
+        device (str, optional): Device to use for computation ('cuda' or 'cpu').
+        model (torch.nn.Module, optional): This is not used for LEAR. It is included for compatibility with the base class.
+    """
+
+    def __init__(self, model_name="wltjr1007/LEAR", device=None, model=None):
```

Comment:
Everything looks great! Could we only, please, rearrange the parameter order for better alignment between all other explainers like this?
```suggestion
    def __init__(self, model=None, model_name: str="wltjr1007/LEAR", device: torch.device=None):
```

Reply:
Thanks. Just comitted your suggestion :)

Reply:
Thank you!